### DISCUSSION TITLE
一个CS和MR资源获取的问题

### DISCUSSION THREAD

**Discussion by Chiloven945** [Meloong-Git/PCL#1408] [提问] [answered]
如果这俩渠道要分开的话那么就没什么事情了，
我猜的是：1.会先尝试从CS获取，这样就导致CS在获取不到的时候MR没有进行获取就一直卡在那
2.两边都会同时获取，但是如果一边没有加载出来那么另一边就不会显示会一直加载
~~（主要是MR我基本上秒访问，CS我要等半天）~~
如果是我说的那样的话我建议要么只有一边加载出来的话提示哪一边没加载出来然后显示那一边的内容，当然我还是建议把渠道分开，可以加个开关决定是否分开

**Comment by RawDiamondMC** [comment]
已有issue，见#1320

**Comment by WForst-Breeze** [comment, marked as answer]
已有 Issue 提及相关问题，此 Discussion 是其进一步论证和说明，请移步 #1320 进行补充，谢谢~
